packages/create-crc-app/templates/src/types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-var */
+import ChromeApi from '@redhat-cloud-services/frontend-components/ChromeApi';
```

Comment:
This should not be necessary :thinking: the types package should be settings this up already.